data/json/mapgen/nested/house_nested.json

```diff
@@ -1546,5 +1546,633 @@
+        "~": { "item": "SUS_bathroom_cabinet", "chance": 40, "repeat": [ 1, 2 ] }
+      },
+      "place_loot": [ { "group": "SUS_hair_drawer", "x": 2, "y": 2, "chance": 30, "repeat": [ 1, 2 ] } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_3x2_WN3",
+    "object": {
+      "mapgensize": [ 4, 4 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        ",}} ",
+        "j,, ",
+        "    ",
+        "  t "
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "liquids": { "T": { "liquid": "water", "amount": [ 0, 50 ] } },
+      "items": {
+        "j": [ { "item": "SUS_bathroom_sink", "chance": 50, "repeat": [ 1, 2 ] }, { "item": "SUS_bathroom_medicine", "chance": 50, "repeat": [ 1, 2 ] }, ],
+        "t": { "item": "SUS_toilet", "chance": 80 }
+      }
+    },
+    "place_monsters": [ { "monster": "GROUP_BLOB", "chance": 70, "x": 2, "y": 2 } ]
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_3x2_WN4",
+    "object": {
+      "mapgensize": [ 4, 4 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        ",~{ ",
+        "j,, ",
+        "    ",
+        "  t "
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "items": {
+        "t": { "item": "SUS_toilet", "chance": 80 },
+        "j": [ { "item": "SUS_bathroom_sink", "chance": 50, "repeat": [ 1, 2 ] }, { "item": "SUS_bathroom_medicine", "chance": 50, "repeat": [ 1, 2 ] }, ],
+        "~": { "item": "SUS_bathroom_cabinet", "chance": 50, "repeat": [ 1, 2 ] }
+      },
+      "place_monsters": [ { "monster": "GROUP_PEST", "chance": 50, "x": 2, "y": 2 } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_3x3_W1",
+    "object": {
+      "mapgensize": [ 3, 3 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "tV}",
+        ",,}",
+        "JjJ"
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "items": {
+        "t": { "item": "SUS_toilet", "chance": 80 },
+        "j": [ { "item": "SUS_bathroom_sink", "chance": 50, "repeat": [ 1, 2 ] }, { "item": "SUS_bathroom_medicine", "chance": 50, "repeat": [ 1, 2 ] }, ],
+        "j": { "item": "SUS_bathroom_sink", "chance": 50, "repeat": [ 1, 2 ] },
+        "V": { "item": "SUS_bathroom_cabinet", "chance": 40, "repeat": [ 1, 2 ] }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_3x3_W2",
+    "object": {
+      "mapgensize": [ 3, 3 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "tVJ",
+        ",,j",
+        "}}J"
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "items": {
+        "t": { "item": "SUS_toilet", "chance": 80 },
```

Comment:
I'm not sure how I feel about the SUS toilet group generating in the same tile as the toilet with all of these. That will make it look like the plunger and toilet brush are literally in the toilet.